e2e/benchmark_test.go

```diff
@@ -0,0 +1,322 @@
+				// 	Enabled: true,
+				// 	ScannersList: []string{
+				// 		"grype",
+				// 		"trivy",
+				// 	},
+				// },
+				// Secrets: secrets.Config{
+				// 	Enabled: true,
+				// 	ScannersList: []string{
+				// 		"gitleaks",
+				// 	},
+				// },
+				// Exploits: exploits.Config{
+				// 	Enabled: true,
+				// 	ScannersList: []string{
+				// 		"exploitdb",
+				// 	},
+				// },
+				// Misconfiguration: misconfigurations.Config{
+				// 	Enabled: true,
+				// 	ScannersList: []string{
+				// 		"lynis",
+				// 		"cisdocker",
+				// 		"fake",
+				// 	},
+				// },
+				// Rootkits: rootkits.Config{
+				// 	Enabled: true,
+				// 	ScannersList: []string{
+				// 		"chkrootkit",
+				// 	},
+				// },
+				// Malware: malware.Config{
+				// 	Enabled: true,
+				// 	ScannersList: []string{
+				// 		"clam",
+				// 		"yara",
+				// 	},
+				// },
+				// InfoFinder: infofinder.Config{
+				// 	Enabled: true,
+				// 	ScannersList: []string{
+				// 		"sshTopology",
+				// 	},
+				// },
+				Plugins: plugins.Config{
+					Enabled:      true,
+					ScannersList: []string{scannerPluginNameKICS},
+					ScannersConfig: plugins.ScannersConfig{
+						scannerPluginNameKICS: config.Config{
+							Name:          scannerPluginNameKICS,
+							ImageName:     cfg.TestEnvConfig.Images.PluginKics,
+							InputDir:      "",
+							ScannerConfig: "",
+						},
+					},
+				},
+			}
+
+			image, cleanup, err := containerrootfs.GetImageWithCleanup(ctx, testImageSourcePath)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			defer cleanup()
+
+			err = containerrootfs.ToDirectory(ctx, image, rootfsPath)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+			scannerConfig.AddInputs(scannercommon.ROOTFS, []string{rootfsPath})
+			scannerConfig.AddInputs(scannercommon.IMAGE, []string{alpineImage})
+
+			input, err := filepath.Abs("./testdata")
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			scannerConfig.AddInputs(scannercommon.ROOTFS, []string{input})
+
+			notifier := &BenchmarkNotifier{
+				started:       make(map[families.FamilyType]time.Time),
+				finished:      make(map[families.FamilyType]time.Time),
+				findingsCount: make(map[families.FamilyType]int),
+				scannerName:   make(map[families.FamilyType]string),
+			}
+
+			errs := scanner.New(scannerConfig).Run(ctx, notifier)
+			gomega.Expect(errs).To(gomega.BeEmpty())
+
+			mdTable, err := notifier.GenerateMarkdownTable()
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+			err = writeMarkdownTableToFile(mdTable)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		})
+	})
+})
+
+func (n *BenchmarkNotifier) GenerateMarkdownTable() (string, error) {
+	rows := [][]string{}
+
+	earliestStartTime := time.Now().UTC()
+	totalFindings := 0
+	for famType, startTime := range n.started {
+		if startTime.Before(earliestStartTime) {
+			earliestStartTime = startTime
+		}
+
+		rows = append(rows, []string{
+			fmt.Sprintf("%s (%s)", famType, n.scannerName[famType]),
```

Comment:
```suggestion
			fmt.Sprintf("%s/%s", famType, n.scannerName[famType]),
```

Reply:
makes it clearer that there is a subtask running, while we can also have a breakdown of the scan task itself, for example:

| Family/Scanner | Start time          | End time            | Findings | Total time |
| -------------- | ------------------- | ------------------- | -------- | ---------- |
| plugins/kics | 2024-07-15 08:44:32 | 2024-07-15 08:44:49 | 23       | 18s        |
| plugins/* | 2024-07-15 08:44:32 | 2024-07-15 08:44:49 | 23       | 18s        |
| malware/yara | 2024-07-15 08:44:32 | 2024-07-15 08:44:49 | 0       | 18s        |
| malware/* | 2024-07-15 08:44:32 | 2024-07-15 08:44:49 | 0       | 18s        |
|   |   |   |         |          |
| _total_ | _2024-07-15 08:44:32_ | _2024-07-15 08:44:49_ | _23_       | _18s_        |

Note: wildcard indicate total time to execute a specific family, `family/scanner` indicate how long specific scanner within a family took.

For family summary, the start time is the time of the earliest family scanner to start, while end time is the time of the last family scanner to finish.

For overall summary, the start and end times are the same, but across all notifications.